### PR TITLE
Adding scaffolding

### PR DESCRIPTION
**Pull request by AlexisMartinez1913**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    width: 100px;
+    text-align: center;
+    
+}
+a{
+    font-size: 24px;
+    color: black;
+    text-decoration: none;
+    
+}
```
Comment (serdnah2) [inline css/style.css]: Please add new line at the end of the file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <nav id="navigation">
+        <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="/about">About</a></li>
+            <li><a href="/Shop">Shop</a></li>
+            <li><a href="/Content">Content</a></li>
+        </ul>
+    </nav>
+</body>
+</html>
```
Comment (SaulAMP) [inline index.html]: Ppor favor añade el espacio al final de el codigo

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+body{
```
Comment (serdnah2) [inline css/style.css]: please add space before of **{**

File: index.html (hunk 2 above)
Comment (serdnah2) [inline index.html]: Please add new line at the end of the file

**Review by serdnah2 — APPROVED** [review]
Approved by Peer

**Review by DilbaniE — COMMENTED** [review]
si, esta aprobado Alexis :)

**Review by MejiaLuigi — APPROVED** [review]
Approved

File: css/style.css (hunk 3 above)
Comment (Jander937) [reply css/style.css]: Approved

**Review by SaulAMP — APPROVED** [review]
SaulAMP les doy permiso

**Pull request merged**